packages/documentation/docs/controls/_breadcrumb_styleguide.md

```diff
@@ -0,0 +1,101 @@
+
+### Content
+
+Breadcrumb items can, but don't have to, include an icon. The icon is positioned before the breadcrumb label. Icons can be included for each item or only for specific items (e.g. the root item).
+
+## Behavior in context
+
+### Population
+
+The breadcrumb should be populated location-based to reflect the hierarchy of the application and the loction of the user in it. The current location should always be included in the breadcrumb.
+
+### Overflow
+
+The breadcrumb control is set to display a limited number of items (default: 9 items). If the number of items exceeds the defined limit, excessive items are hidden within a dropdown menu at the beginning of the path. The dropdown menu is triggered by pressing the respective item. The truncation is visualized with a ellipsis.
+
+The overflow behaviour can also be triggered if the available space does not allow the complete display of the breadcrumb in one line.
+
+Truncation is applied to individual breadcrumb items if the maximum width of the breadcrumb item is exceeded. The label name is truncated with an ellipsis.
+
+<!-- add example overflow & truncation-->
+
+### Placement
+
+Breadcrumbs are placed at the top left side of the page/ content area. They are positioned below the header and above the page title.
+
+## States
+
+### Interactive item
+
+| State   | Description                                                                          |
+| ------- | ------------------------------------------------------------------------------------ |
+| Hover   | The hover state is indicated by font and fill color change.                          |
+| Active  | The active state is indicated by font and fill color change. Equal to pressed state. |
+| Focused | The focused state is indicated by a focus frame.                                     |
+
+### Not interactive item
+
+No interactions available.
+
+## Anatomy
+
+### Main elements
+
+The breadcrumb consists of at least one item. Each item contains a breadcrumb label. All but the last item in the breadcrumb path are always followed by a chevron icon.
+
+### Sizing
+
+- The breadcrumb has a fix height defined by breadcrumb item height.
+- The breadcrumb has a maximum width defined by the content area width. Exceeding the width will trigger truncation.
+- The breadcrumb item has a fixed height for single-line text entries.
+- The breadcrumb item width depends on the content and is limited to a maximum width of 14 rem.
+
+### Spacing
+
+Between breadcrumb items, a space of 0.25 rem is introduced.
+
+## Dos and don'ts
+
+- Don't use breadcrumbs to display a multistep process. Use a workflow control instead.
```

Comment:
link the last sentence to the workflow chapter? (same for the releated patterns)